stage1/init/common/pod.go

```diff
@@ -794,3 +794,22 @@ func protectKernelTunables(opts []*unit.UnitOption, appName types.ACName, system
 
 	return opts
 }
+
+// generateWritablePaths appends the list of options required to mark
+// a rootfs RO and specific paths as RW, according to runtime options.
+func generateWritablePaths(opts []*unit.UnitOption, podAbsRoot string, runApp *schema.RuntimeApp, vols map[types.ACName]types.Volume, mounts []mountWrapper) ([]*unit.UnitOption, error) {
```

Comment:
Hmm .. since we are moving more unit generation logic into `units.go`, I don't see this function to be in `pod.go`